cli/sawtooth_cli/rest_client.py

```diff
@@ -40,19 +40,44 @@ def get_leaf(self, address, head=None):
         queries = RestClient._remove_nones(head=head)
         return self._get('/state/' + address, queries)
 
+    def send_batches(self, batch_list):
+        """Sends a list of batches to the validator.
+
+        Args:
+            batch_list (:obj:`BatchList`): the list of batches
```

Comment:
Not meaning for you to fix anything, but I have seen several ways in which doc strings have been formatted.  Seems like we should settle on a "sawtooth standard"